common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationServiceImpl.java

```diff
@@ -46,6 +46,7 @@
     protected static final String COULD_NOT_MINIFY_RESOURCES_RETURNED_UNMINIFIED_BYTES = "Could not minify resources, returned unminified bytes";
     public static String CSS_TYPE = "css";
     public static String JS_TYPE = "js";
+    public static String JS_MIN = ".min.js";
 
```

Comment:
Should check .min.css as well since there's no point in minifying it as well even if YUICompressor doesn't have an issue with it